src/components/panel/git/RepositoryAddModal.tsx

```
@@ -0,0 +1,88 @@
+  };
+
+  return (
+    <Modal
+      title="레포지토리 추가"
+      onClose={handleClose}
+      primaryLabel={isPending ? "추가 중..." : "추가"}
+      onPrimaryClick={handlePrimary}
+      isPrimaryDisabled={isPending || !name.trim() || !url.trim()}
+    >
+      <div className="flex flex-col gap-1">
+        <label className="typo-label text-(--color-text-secondary)">이름</label>
```

Biome (2.4.15)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)